lib/tasks/sumofus.rake

```diff
@@ -0,0 +1,51 @@
+require 'open-uri'
+
+namespace :sumofus do
+  desc 'Import legacy actions into your database from a specified file'
+  task seed_legacy_actions: :environment do
+    if ARGV[1].nil?
+      abort('Requires a valid url to a file containing the legacy actions to seed.')
+    end
+
+    if ARGV[2].nil?
+      abort('Requires a valid url to a file containing a default header image to attach to the page.')
+    end
+
+    file_handle = open(ARGV[1])
+    data = file_handle.read
+    file_handle.close
+
+    image_handle = open(ARGV[2])
+
+    parsed_data = JSON.load data
+    count = 0
+    layout_id = LiquidLayout.where(title: 'Petition With Small Image').first.id
+    parsed_data.each_pair do |k, entry|
```

Comment:
I was thinking about this this afternoon and I think we should revisit the logging from this script. @EricBoersma you make a great point that we should print at the end the number of pages we saved. The problem is that we aren't actually tracking the number of pages that actually got successfully saved. All these calls to `.save` could just as easily return false. Also, using the `!` version of `find_or_create_by` will cause the script to raise any validation error and terminate the loop partway through, which seems not unlikely to me since we validate that the page title is unique. The case where the page save returns false isn't handled by the code for the plugins, which could cause it to through `NoMethodErrors` because `thermometer` and `petition_form` are nil, or it might still leave orphaned plugins lying around the database. 

Specifically, I think that we should be handling the cases where any of the three `.save` calls return false, or doing something to this effect:
```
page = Page.find_or_create_by(title: title, liquid_layout_id: layout_id) do |page|
  page.language_id = Language.where(code: entry['language']).first.id
  page.active = true
  page.slug = entry['slug']
  page.content = entry['page_content'].gsub(/(?:\n\r?|\r\n?)/, '<br>')
end
if page.errors.size > 1
  failed_pages.push[(page.errors, entry)]
  next
end
```
and do something similar to log failed updates to thermometers and petitions. This way we won't raise any validation errors, and at the end we can print a true count of how many pages were added to the database, plus errors and source data for all the failures.